healthd/healthd_mode_charger.cpp

```diff
@@ -97,7 +101,10 @@ char* locale;
 #define LOGW(x...) KLOG_WARNING("charger", x);
 #define LOGV(x...) KLOG_DEBUG("charger", x);
 
-#define MIN_BATTERY_FOR_BOOT 10
```

Comment:
#define MIN_BATTERY_FOR_BOOT 8

Reply:
Prefer to leave it since there is a potential unintended misuse of the macro somwhere.